src/observeProps.js

```diff
@@ -1,5 +1,11 @@
 import React, { PureComponent } from 'react';
-import { Observable, Observer, Subject } from 'rx-lite';
+//* using exact import paths for rxjs significantly reduces bundle size
+import { Subject } from 'rxjs/internal/Subject';
```

Comment:
@azhalkouski, are u sure we need to import from /internal/ ? Can't we do like that
`import { map } from 'rxjs/operators/map'`?